src/main/java/core/controller/OrderController.java

```diff
@@ -31,15 +33,19 @@ public OrderController(OrderService orderService,
     }
     
     @GetMapping("/orders")
-    public List<OrderResponseDto> getUserOrders(@RequestParam Long userId) {
-        return orderService.getOrdersHistory(userService.get(userId)).stream()
+    public List<OrderResponseDto> getUserOrders(Authentication authentication) {
+        UserDetails details = (UserDetails) authentication.getPrincipal();
+        User user = userService.findByEmail(details.getUsername()).get();
+        return orderService.getOrdersHistory(userService.get(user.getId())).stream()
```

Comment:
```suggestion
        return orderService.getOrdersHistory(userService.get(user.getId())).stream()
```
use method findByEmail()

Reply:
I use it in the line above. But Forgotten to put it on line 39)  Thanks!